inc/saifdb.h

```diff
@@ -57,9 +70,15 @@ typedef struct _sai_fdb_entry_t
     /** Mac address */
     sai_mac_t mac_address;
 
-    /** Vlan ID */
+    /** Bridge type */
+    sai_fdb_entry_bridge_type_t bridge_type;
+
+    /** Vlan ID. Valid for .1Q */
     sai_vlan_id_t vlan_id;
 
+    /** Bridge ID. Valid for .1D */
+    sai_object_id_t bridge_id;
```

Comment:
this value is conditional based on bridge_type which makes this a lot harder to maintain in metadata